cmd/tailscale/cli/web.html

```diff
@@ -99,13 +99,13 @@ <h3 class="text-3xl font-semibold mb-3">Log in</h3>
 		<a href="#" class="mb-4 link font-medium js-loginButton" target="_blank">Reauthenticate</a>
 	</div>
 	{{ if .IsSynology }}
+		{{ if not .TUNMode }}
```

Comment:
and if it is? where'd that text go?

also, please adjust the git title to say something about Synology and web

Reply:
Discussed with @mayakacz offline and she suggested removing entirely, because calling it out in an alert box didn't make sense when it's a success case, and leaving it up by itself underneath reauth/advertise links looked (imo) messy and didn't follow a clear content hierarchy 

<img width="527" alt="Screen Shot 2023-01-24 at 17 55 02" src="https://user-images.githubusercontent.com/582683/214645948-bd451dad-86b8-4b8f-8110-ebd66d0419f0.png">

Reply:
I want something there for debugging with users.

If nothing's there then we don't know whether they're just using an old version that didn't say anything.

Reply:
But yes I can leave that to a separate PR